reflection/reflect.go

```diff
@@ -35,6 +35,36 @@ func GetTestMutex(t *testing.T) *sync.RWMutex {
 	return nil
 }
 
+func GetHelpersMap(t *testing.T) map[string]struct{} {
+	t.Helper() // Ensure map creation before accessing it
+	mu := GetTestMutex(t)
+	if mu != nil {
+		mu.Lock()
+		defer mu.Unlock()
+	}
+	if pointer, err := GetFieldPointerOf(t, "helpers"); err == nil {
+		return *(*map[string]struct{})(pointer)
+	}
```

Comment:
This lines can be included in `AddToHelpersMap` func directly (line 62), to avoid the retrieve of the mutex twice.

Reply:
Also add the `t.Helper` in the `AddToHelpersMap` func and remove this `GetHelpersMap`